cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -689,6 +696,58 @@ def test_output_non_course_author(self):
         resp = client.get(reverse_course_url('export_output_handler', self.course.id))
         self.assertEqual(resp.status_code, 403)
 
+    def _mock_artifact(self, type=None, file_url=None):
```

Comment:
Quality fails because: https://build.testeng.edx.org/job/edx-platform-quality-flow-pr/5872/artifact/reports/diff_quality/diff_quality_pylint.html

Reply:
Thanks! Fixed in 35d8472.